packages/atomic/src/templates/default.html

```diff
@@ -51,9 +56,10 @@
   </div>
 
   <div class="content-container">
-    <atomic-result-link></atomic-result-link>
-
-    <!-- TODO: add "date type" for atomic-result-value -->
+    <div class="title-container">
+      <atomic-result-link></atomic-result-link>
+      <atomic-result-value value="date" format="date"></atomic-result-value>
```

Comment:
I think `field` or `property` or `key` would be a better name for the `value` prop, since the string is not the actual value.